web/scripts/pretty-docker-logs.js

```diff
@@ -0,0 +1,134 @@
+const chk = require("chalk")
```

Comment:
> Made docker logs very pretty if you're running pnpm dev. This is accomplished by an absurd log parsing abomination that I made in web/scripts/pretty-docker-logs.js. If this ever breaks something, just get rid of it. It does make the logs much nicer to read though. If you need to view the raw logs, you can run pnpm serve.

You should add some variant of this disclaimer to the file.

Reply:
Oh yeah, will do.